src/Symfony/Contracts/Translation/TranslatorTrait.php

```diff
@@ -115,6 +115,10 @@ public function trans($id, array $parameters = [], $domain = null, $locale = nul
                 return strtr($standardRules[0], $parameters);
             }
 
+            if (false === $behaveSilently) {
+                return;
```

Comment:
shouldnt we always return a string, per contract? Also i tend to believe it's silentness/debug mode is a semantic of a (debug) translator, instead of handling it per trans() call.

Reply:
an empty string then ? (see comment below :p)